src/api/financial/financial.js

```diff
@@ -0,0 +1,46 @@
+import fetch from 'node-fetch';
+import { determineError } from '../../services/errors';
+import http from '../../services/http';
+
+/**
+ * Class dealing with the /financial-actions api endpoint
+ *
+ * @export
+ * @class Financial
+ */
+export default class Financial {
+    constructor(config) {
+        this.config = config;
+    }
+
+    /**
+     * Returns the list of financial actions and their details.
+     *
+     * @memberof Financial
+     * @param {Object} parameters Parameters.
+     * @return {Promise<Object>} A promise to the Financial Actions response.
+     */
+    async query(parameters) {
+        try {
+            let url = `${this.config.host}/financial-actions`;
+
+            if (parameters) {
+                const queryString = Object.keys(parameters)
+                    .map((key) => `${key}=${parameters[key]}`)
+                    .join('&');
+                url += `?${queryString}`;
+            }
+
+            const response = await http(fetch, this.config, {

```

Comment:
Can we use `getHandler` from `payments.js`

Reply:
I don't know if I like the idea of using something from `payments.js` here in the financial client. But maybe we can discuss if `getHandler` should be like a utils function or maybe even refactor the `http.js` to be similar to what we have in the other SDKs. WDYT?

Reply:
We could extract this common function.

Reply:
This change is going to be implemented in a future sprint. The process will be tracked with this [task](https://checkout.atlassian.net/browse/INT-1152)